### ISSUE TITLE
Fix the way we're generating certificates on our test deployments

### ISSUE THREAD

**Issue by svyatonik** [paritytech/parity-bridges-common#1396]
Right now we're hitting 5 certs/week limit (https://letsencrypt.org/docs/rate-limits/) when generating certificates for our test deployments nodes:
```
Create new order error. Le_OrderFinalize not found. {
  "type": "urn:ietf:params:acme:error:rateLimited",
  "detail": "Error creating new order :: too many certificates (5) already issued for this exact set of domains in the last 168 hours: ui.brucke.link: see https://letsencrypt.org/docs/rate-limits/",
  "status": 429
}
```

At first glance, possible solutions are:
1) actually store generated certificates (proper way imo);
2) do restarts less often (e.g. every other day).

**Comment by serban300** [comment]
According to [this](https://github.com/nginx-proxy/acme-companion/issues/789) and [this](https://github.com/nginx-proxy/acme-companion/issues/510) issues we might need to add a volume:
```
- letsencrypt-acme:/etc/acme.sh
```
both for the `nginx-proxy` and the `nginx-proxy-le` containers in order to persist the SSL certificates.

**Comment by serban300** [comment]
Merged #1554 that should fix this. However keeping this opened for a while until the current rate-limiting window expires and we can check that the issue is actually fixed.

**Comment by serban300** [comment]
The issue doesn't seem to reproduce and from the logs it seems that the certificates are persisted and aren't renewed on every restart:

```
Creating/renewal grafana.millau.brucke.link certificates... (grafana.millau.brucke.link grafana.rialto.brucke.link)
[Mon Aug 29 05:01:52 UTC 2022] Domains not changed.
[Mon Aug 29 05:01:52 UTC 2022] Skip, Next renewal time is: Sun Oct 23 12:03:35 UTC 2022
[Mon Aug 29 05:01:52 UTC 2022] Add '--force' to force to renew.
```

Closing the issue
